01_prepare_host.sh

```diff
@@ -99,7 +107,7 @@ if ! id "$USER" | grep -q libvirt; then
 fi
 
 # Start image downloader container
-sudo "${CONTAINER_RUNTIME}" run -d --net host --privileged --name ipa-downloader ${POD_NAME} \
+sudo "${CONTAINER_RUNTIME}" run -d --net host --privileged --name ipa-downloader "${POD_NAME}" \
```

Comment:
don't change this. POD_NAME contains "--pod abc" so if you put quotes it is considered as one string and fails when running with podman.

Reply:
Fixed